translations/es/example-notebook.ipynb

```diff
@@ -451,6 +455,7 @@
    ]
```

Comment:
need include the section "Use plotting for visual context" https://github.com/OscarSantosMu/codespaces-teaching-template-py/blob/main/example-notebook.ipynb

Reply:
Can you provide more details? The only update I did in the [translations/es/example-notebook.ipynb](https://github.com/education/codespaces-teaching-template-py/pull/41/files/2225b078826cee9a64e577de2fd9a810756b92eb#diff-fd259aaa37fc2a64cf08eec1233ffcd3efecf51acfa0901db8f77171cfb23dbe) was a single rephrase at the "Operaciones básicas de Pandas" paragraph.

Reply:
Hi @OscarSantosMu include a new section named "Use plotting for visual context". You can check the new section and compare with your file (I'm linking the notebook in english): https://github.com/OscarSantosMu/codespaces-teaching-template-py/blob/main/example-notebook.ipynb

Reply:
I do not get it, the section is already there for me.

![image](https://github.com/education/codespaces-teaching-template-py/assets/38818848/973c4904-eda9-480c-a70f-828363e4838a)

left (spanish section) - right (linked english section)

And the section is also mentioned at the Learning Objectives (Objetivos de aprendizaje) as well.

Reply:
This looks OK to me @cyz . Thanks @OscarSantosMu

Reply:
thank you @alfredodeza for you review and @OscarSantosMu :) all good!